projects/ngx-feature-toggle/src/lib/ngx-feature-toggle-route-guard.router.ts

```diff
@@ -29,7 +29,7 @@ export class NgxFeatureToggleRouteGuard implements CanActivateChild, CanLoad, Ca
       toggle[0] === '!' ? !isOn(toggle.replace('!', '')) : isOn(toggle),
     );
 
-    if (!hasAllTogglesOn && route.data.redirectTo) {
+    if (!hasAllTogglesOn && route.data.redirectTo != null) {
```

Comment:
It can be even better if check for the value itself using `!==` instead of `!=`

```suggestion
    if (!hasAllTogglesOn && typeof route.data.redirectTo !== null) {
```

Reply:
The only problem with this is that if redirectTo = undefined, the expression would evaluate to true, which could cause problems.

Reply:
If you'd like to be more explicit and stick to strict equality comparison, we could change it to `!hasAllTogglesOn && route.data.redirectTo !== null && route.data.redirectTo !== undefined)`

Reply:
Or even better! At the moment, `redirectTo` is accepting only strings as a valid value, so we can have the code checking only this case to avoid any caveats like that. So that we avoid several misconfiguration scenarios

```suggestion
    if (!hasAllTogglesOn && typeof route.data.redirectTo === 'string') {
```